sw-emulator/lib/periph/src/mailbox.rs

```diff
@@ -194,7 +194,7 @@ pub enum MailboxRequester {
 impl From<u32> for MailboxRequester {
     fn from(value: u32) -> Self {
         match value {
-            0 => MailboxRequester::Caliptra,
+            0xFFFF_FFFF => MailboxRequester::Caliptra,
```

Comment:
I thought this user was no longer valid?

Reply:
It's not valid for the SoC to use it because it's reserved for Caliptra's own use.